ast/ast.go

```diff
@@ -53,32 +53,36 @@ type Nodes []Node
 // NodeBase holds fields common to all node types.
 type NodeBase struct {
 	LocRange LocationRange
+	Fodder   Fodder
```

Comment:
What is this main Fodder that goes in every node? Perhaps a comment here would be nice.

Reply:
Done.